config/test_suites/merge_pull_request_openshift_ansible.yml

```diff
@@ -0,0 +1,4 @@
+---
+children:
+  - "test_branch_origin_extended_conformance_install_with_status_check"
```

Comment:
These should come from `test_pull_request_openshift_ansible...` not Origin branch